src/constants/urls.ts

```diff
@@ -0,0 +1,4 @@
+export enum URLS {
+	// ----- WILDERWORLD ----- //
+	WILDERWORLD = 'https://www.wilderworld.com/',
+}
```

Comment:
@colbr @padala @domw30 @akshatmittal We will start using global constants for external URLs instead of hard coding in each component